sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_client_async.py

```diff
@@ -678,3 +679,36 @@ def get_subscription_receiver(
             )
         self._handlers.add(handler)
         return handler
+
+    def get_management_operation_client(self,
+            entity_name: str,
```

Comment:
entity_name because what if we want to do a mgmt operation on a topic...
could also do queue_name, topic_name as kwargs